src/main/java/mealplan/MealplanInteractor.java

```diff
@@ -0,0 +1,59 @@
+package mealplan;
+import entities.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import recipe.*;
+
+public class MealplanInteractor implements MealplanInputBoundary{
+    String filePath = "C:\\Users\\lucia\\Desktop";
```

Comment:
Once again, you should probably replace the filename so that the code works for everyone else.